ui/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "Canary Checker",
```

Comment:
We need to update these and other references from canary checker to Incident Commander